Brokerages/GDAX/GDAXBrokerage.Messaging.cs

```diff
@@ -257,30 +273,33 @@ private void OrderDone(string data)
             }
 
             //is this our order?
-            var cached = CachedOrderIDs.Where(o => o.Value.BrokerId.Contains(message.OrderId));
+            var cached = CachedOrderIDs.Where(o => o.Value.BrokerId.Contains(message.OrderId)).ToList();
```

Comment:
This whole `CachedOrderIDs` collection is somewhat bothersome to me. LEAN provides a type, `IOrderProvider` that will fetch orders. My concern with this approach is we now have multiple stores of active orders (transaction handler and brokerage). I realize `CachedOrderIDs` is existing code, but once again, just thinking out loud.